src/components/Facility/PatientNoteCard.tsx

```diff
@@ -213,7 +213,7 @@ const PatientNoteCard = ({
                 </div>
               </div>
             ) : (
-              <div className="text-sm text-secondary-700 whitespace-pre-wrap">
+              <div className="text-sm text-secondary-700 break-all">
```

Comment:
this doesn't respect new lines present in the message content right?

Reply:
working fine 
<img width="405" alt="Screenshot 2024-12-30 at 12 17 04 PM" src="https://github.com/user-attachments/assets/4bf5d97f-7cf6-40d4-9048-ad7cd49fb1d4" />
<img width="1088" alt="Screenshot 2024-12-30 at 12 17 32 PM" src="https://github.com/user-attachments/assets/318edb81-db67-44f4-b50c-59b1b4ab61ee" />

Reply:
i don't know any word in dictionary that's that long. this is not needed

Reply:
Previously 
<img width="1187" alt="Screenshot 2024-12-30 at 12 19 11 PM" src="https://github.com/user-attachments/assets/12f1da4d-759e-4827-b4e2-a699a995f691" />
<img width="435" alt="Screenshot 2024-12-30 at 12 18 58 PM" src="https://github.com/user-attachments/assets/245d60a1-f93d-459e-b9c1-4bb2ea343993" />

Reply:
![image](https://github.com/user-attachments/assets/b2b8c819-d341-41b4-90e9-51adef602576)